docutils-stubs/writers/manpage.pyi

```diff
@@ -6,7 +6,9 @@ from docutils import nodes, SettingsSpec, writers
 from docutils.parsers.rst.directives.html import MetaBody
 from docutils.writers import manpage
 from types import ModuleType
-from typing import Any, Callable, Dict, List, Optional, Pattern, Tuple, Type
+from typing import Any, Callable, Dict, List, Optional, Pattern, Tuple, Type, TypeVar
+
+N_co = TypeVar('N_co', bound=nodes.Node, covariant=True)
```

Comment:
TypeVar `N_co` is not used in manpage.pyi file.
Would you aim at using this TypeVar in order to fix the type of `Translator.colspec`?

https://github.com/tk0miya/docutils-stubs/blob/60eada677c6c518f871ffab391e9155b6e6e5673/docutils-stubs/writers/manpage.pyi#L47

Reply:
oh, sorry. It was mistake. removed.